jekyll/_cci2/bitbucket-data-center-integration.adoc

```diff
@@ -0,0 +1,252 @@
+
+Integrate with CircleCI to access features for automated software delivery:
+
+* xref:workflows#workflows-configuration-examples[Workflow orchestration]
+* xref:optimizations#[Data, compute, and time optimization]
+* Control over xref:security#[security] and xref:contexts#[permissions], and org-level control with xref:config-policy-management-overview#[config policies]
+* xref:executor-intro#[Managed execution environments] (Docker, Linux VM, macOS, Windows, Arm VM, GPU), plus xref:runner-overview#[self-hosted runners]
+* xref:test#[Test integrations], and xref:parallelism-faster-jobs#[test-splitting] to reduce length of the testing phase of your pipelines
+* xref:orb-intro#[Packaged config (orbs)] to simplify integration with many third party platforms, services, language tools etc.
+* Developer tools: xref:local-cli#[CircleCI CLI], xref:api-intro#[APIs], and xref:vs-code-extension-overview#[VS Code extension], xref:circleci-config-sdk#[config SDK]
+
+== Steps to integrate Bitbucket Data Center projects with CircleCI
+
+=== Prerequisites
+
+* Follow the steps on the xref:first-steps#[Sign up and try CircleCI] page to create a CircleCI account, create a new CircleCI organization. Alternatively you can use an existing organization.
+
+* Follow the steps on the xref:create-project#create-a-project[Create a project] page to create a project in CircleCI and set up an integration with your Bitbucket Data Center project.
+
+=== Integrate a project with CircleCI
+
+. **Set up your integration with Bitbucket Data Center if you have not already**:
+.. In the CircleCI web app, select **Organization Settings** in the sidebar.
+.. Select **Integrations** in the sidebar.
+.. Select btn:[Set Up Integration] next to Bitbucket Data Center.
+.. In the modal, enter your Bitbucket Data Center instance URL and your known_hosts. For guidance on generating known_hosts, see <<known-hosts>>.
+.. Select btn:[Set Up Integration].
+.. Select the back arrow to exit **Organization Settings**.
+
+. For the next sections you will need to **create a project HTTP access token** with the project admin scope. Create this under Projects in Bitbucket. For more information, see the link:https://confluence.atlassian.com/bitbucketserver/http-access-tokens-939515499.html[Bitbucket docs]. Copy the token somewhere safe, you will need to enter it when creating pipelines and triggers in CircleCI.
+
+. **Set up a pipeline**:
+.. Select **Project Settings**.
+.. Select **Pipelines** in the sidebar.
+.. Select **Set Up Pipeline**. Fill in the required fields:
+** Give your pipeline a descriptive name.
+** Select **Bitbucket Data Center** under Config Source.
+** Provide the project HTTP access token you created in the previous step. Select btn:[Connect].
+** Select your repository and branch.
+** Enter `.circleci/config.yml` in the Config File Path field.
```

Comment:
They will want to enter the path to wherever their config file is hosted, in most cases it'll be `.circleci/config.yml` but could exist anywhere in the repo -- can we adjust this to maybe something like:

>Enter the path to where your CircleCI config file lives within your repository, if you've followed our docs, this likely will be `.circleci/config.yml`.